esptfa_arima/Test_Management/views.py

```
@@ -612,164 +828,212 @@ def get_context_data(self, **kwargs):
-                        gap_to_passing=ExpressionWrapper(
-                            F('passing_threshold') - F('score'),
-                            output_field=FloatField()
-                        ),
-                    )
+                    analysis_document=document
+                ).annotate(
+                    gap_to_passing=ExpressionWrapper(
+                        F("passing_threshold") - F("score"), output_field=FloatField()
+                    ),
+                )
                 for prediction in context["predictions"]:
```

Ruff (0.14.14)

[warning] Loop control variable `prediction` not used within loop body

(B007)